spec/features/picture_feature_spec.rb

```diff
@@ -0,0 +1,67 @@
+require 'rails_helper'
+
+feature 'pictures' do
+  context 'no pictures have been added' do
+    scenario 'should display a prompt to add a picture' do
+      visit '/pictures'
+      expect(page).to have_content 'No pictures yet'
+      expect(page).to have_link 'Add a picture'
+    end
+  end
+
+  context 'picture caption have been added' do
+    before do
+      Picture.create(caption: 'holiday snap')
+    end
+
+    scenario 'display captions' do
+      visit '/pictures'
+      expect(page).to have_content('holiday snap')
+      expect(page).not_to have_content('No pictures yet')
+    end
+  end
+
+  before do
+    visit '/'
+    click_link 'Sign up'
+    fill_in 'Email', with: 'test@email.com'
+    fill_in 'Password', with: 'randomness'
+    fill_in 'Password confirmation', with: 'randomness'
+    click_button 'Sign up'
+  end
```

Comment:
Placing this halfway through the tests is a bit odd - pop it in a context to scope it to subsequent tests